views/main/test.ejs

```diff
@@ -1 +1,4 @@
 <div>TEST</div>
+<div id="test"></div>
```

Comment:
What is this for?

Reply:
Used this to check what countries were missing flag icons. Is kinda leftover code, but this entire page is going away once we reach deployment either way.